pkg/k8s/deployments/translate.go

```diff
@@ -219,6 +220,29 @@ func TranslateDevContainer(c *apiv1.Container, rule *model.TranslationRule) {
 	TranslateContainerSecurityContext(c, rule.SecurityContext)
 }
 
+func TranslateInitContainer(initContainer *model.InitContainer) {
+	if initContainer.Image == "" {
```

Comment:
What's going to happen if the init container provided by the user doesn't have the binaries we need?  Can we check that we fail gracefully in that scenario?

Reply:
Also, what happens if the user has an older bin image here? How will we troubleshoot that?

Reply:
we can add a binary to the `okteto/bin` image that returns the version of the bin image, and validate the version as part of the clean command:
https://github.com/okteto/okteto/blob/master/cmd/up/up.go#L899